sw/host/cryptotest/testvectors/parsers/nist_cavp_aes_gcm_parser.py

```diff
@@ -30,19 +36,19 @@ def parse_testcases(args) -> None:
             "iv": str_to_byte_array(test_vec["IV"]),
             "ciphertext": str_to_byte_array(test_vec["CT"]),
             "plaintext": str_to_byte_array(test_vec["PT"]) if "PT" in test_vec else [],
-            "result": True if "PT" in test_vec else False,
+            "result": decrypt_fail_check(args.operation.lower(), test_vec),
```

Comment:
Nit: this could be one-lined: `"result": operation != "decrypt" or "PT" in test_vec`

Reply:
Done